### PR TITLE
Merge the latest changes from development into master

### PR DESCRIPTION
**Pull request by David-Corbett**

### CODE REVIEW

File: imagemin.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const GIFImages = 'src/images/**/*.gif';
 const output = 'images';
 
 const imageminJpegtran = require('imagemin-jpegtran');
-const imageminJpegoptim = require('imagemin-jpegoptim');
+// const imageminJpegoptim = require('imagemin-jpegoptim');
```
Comment (omahm) [inline imagemin.js]: Ideally, any commented out code should be removed.

File: src/js/toggle-btn-script.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+   * by default, can be changed by adding a `data-toggle-length` attribute to
+   * the parent wrapper where the `.toggle-list` class is declared.
+   */
+  // eslint-disable-next-line one-var,no-var
+  var $relatedArticleList = $('.toggle-list ul'),
+    maxRelated =
+      parseInt($('.toggle-list').attr('data-toggle-length'), 10) || 7,
+    relatedTotal = $relatedArticleList.children('li').length,
+    // eslint-disable-next-line
+    relatedToggleText = 'Show ' + (relatedTotal - maxRelated) + ' more';
```
Comment (omahm) [inline src/js/toggle-btn-script.js]: How are we going to deal with translations?

**Review by omahm — COMMENTED** [review]
Looks good. just a couple of comments.

File: imagemin.js (hunk 1 above)
Comment (David-Corbett) [reply imagemin.js]: uncommented the line - was added for debugging and not reversed

File: src/js/toggle-btn-script.js (hunk 2 above)
Comment (David-Corbett) [reply src/js/toggle-btn-script.js]: have changed this to use the Drupal.t() functionality

**Pull request merged**